mqtt3/1.0.0/modules/ROOT/pages/mqtt3-connector-cluster.adoc

```diff
@@ -0,0 +1,13 @@
+= Understand Deploying MQTT3 Connector Apps in a Cluster
+
+When an MQTT broker establishes a connection with a client, the broker identifies the client by its client ID. If the broker establishes a connection with a client whose client ID is `smart-bentley-123`, and then the broker receives a new connection with the same client ID, the broker closes the previous connection, assuming it is stale and takes on the new connection. This mechanism is known as a client take-over. MQTT brokers employ the mechanism because in certain situations TCP connections can reach a half-open or invalid state, perhaps due to a crash in one of the communication ends, which is undesirable. The broker assumes the new incoming connection as the valid one, closes the older connection, and continues using the newly established connection.
+
+Because of this mechanism, Mule runtime engine (Mule) applications that use MQTT3 Connector deployed in a cluster result in a different client ID generated for every configuration in each deployed instance. Even when the specified client ID is the same in all the configurations. Therefore, you do not need to concern with defining a random or dynamic ID to avoid client take-overs between the nodes.
```

Comment:
```suggestion
Because of this mechanism, Mule runtime engine (Mule) applications that use MQTT3 Connector deployed in a cluster generate a unique client ID for every configuration in each deployed instance. Even when the specified client ID is the same in all the configurations. Therefore, you do not need to concern yourself with defining a random or dynamic ID to avoid client take-overs between the nodes.
```